spinalcordtoolbox/metadata.py

```diff
@@ -224,3 +224,17 @@ def get_indiv_label_names(directory):
 
     return tuple([_name for (_id, _name, _file) in il._indiv_labels])
 
+def get_indiv_label_ids(directory):
```

Comment:
Not sure if we want to create a new function or to refactor the function `get_indiv_label_names`, which could return two tuples (one for names, one for IDs).

Reply:
Ok for the new function, but ideally we should output an object with the id, name and file name. That would minimize the code

Reply:
Agree! Done :-)